### PR TITLE
First problems

### PR DESCRIPTION
**Pull request by Vamonosraza**

### CODE REVIEW

File: two_sum.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from typing import List
+class Solution:
```
Comment (chusteven) [inline two_sum.py:2]:
```suggestion
from typing import List


class Solution:
```

So one thing you'll see a lot once you start working is comments like these that start with the word "nit" -- they're intended to be clear that the reviewer (me) is just being nitpicky about something. They don't expect you to take their suggestion, but they would like you to :) Basically it's a token of goodwill, while expressing their preference (as it turns out, there are A LOT of preferences and opinions in coding :P )

Nit: between imports and code, there should be two spaces

File: two_sum.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    def twoSum(self, nums: List[int], target: int) -> List[int]:
+        for i, num1 in enumerate(nums):
+            for j,num2 in enumerate(nums):
```
Comment (chusteven) [inline two_sum.py:5]:
```suggestion
            for j, num2 in enumerate(nums):
```

Nit

File: two_sum.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+                if num1 + num2 == target:
+                    return [i, j]
+
+solutions = Solution()
+answer = solutions.twoSum([2,7,11,15],9)
+print(answer)
+answer = solutions.twoSum([3,2,4],6)
+print(answer)
+answer = solutions.twoSum([3,3],6)
+print(answer)
```
Comment (chusteven) [inline two_sum.py:17]:
```suggestion
def main() -> None:
    solutions = Solution()
    answer = solutions.twoSum([2,7,11,15],9)
    print(answer)
    answer = solutions.twoSum([3,2,4],6)
    print(answer)
    answer = solutions.twoSum([3,3],6)
    print(answer)
    
if __name__ == "__main__":
    main()
```

Nit: this is the usual way of starting Python files ; you can read about it [here](https://www.freecodecamp.org/news/if-name-main-python-example/) or just take my word for it :P

**Review by chusteven — APPROVED** [review]
A few things:

1/ See if you can install Black on your VSCode -- that makes formatting your code a lot easier; just google "Black VSCode Windows" or something

2/ See if you can install Pylance on your VSCode -- that makes finding syntax errors in your code a lot faster; google it :)

3/ Remember how we talked about "runtime complexity"? Don't worry about it too much for now; but below I'll add another solution; think about it on your own time why it runs faster than yours (remember yours had 16 iterations for an input of length 4; whereas this one only has 8 iterations total!) (actually there's an even sneakier way to get just 4 iterations total 🤯  but we can save that for later ;) )

```
from typing import Dict
from typing import List


class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashmap: Dict[int, int] = {}
        for i, num in enumerate(nums):
            hashmap[num] = i
        for i, num in enumerate(nums):
            complement = target - num
            if complement in hashmap and hashmap[complement] != i:
                return [i, hashmap[complement]]
```

File: two_sum.py (hunk 1 above)
Comment (Vamonosraza) [reply two_sum.py:2]: Cool, I do not mind. I understand they are coming from a good place and I will these Nit's very helpful. Please keep them coming 👍🏽

**Pull request merged**